src/components/expression/expression-renderer.js

```diff
@@ -452,6 +513,75 @@ genotet.ExpressionRenderer.prototype.drawMatrixConditionLabels_ = function() {
 };
 
 /**
+ * Renders the expression heatmap gradient.
+ * @private
+ */
+genotet.ExpressionRenderer.prototype.drawHeatmapGradient_ = function() {
+  var heatmapData = this.data.matrix;
+  var scaleMin = 0;
+  var scaleMax = 0;
+  if (this.data.options.autoScaleGradient) {
+    scaleMin = heatmapData.valueMin;
+    scaleMax = heatmapData.valueMax;
+  } else {
+    scaleMin = heatmapData.allValueMin;
+    scaleMax = heatmapData.allValueMax;
+  }
+  scaleMin = scaleMin.toFixed(2);
+  scaleMax = scaleMax.toFixed(2);
+  var marginLeft = scaleMin.toString().length *
+    this.CONDITION_LABEL_HEIGHT_FACTOR;
+  var marginRight = scaleMax.toString().length *
+    this.CONDITION_LABEL_HEIGHT_FACTOR;
+
+  var transformLeft = 0;
+  var transformTop = this.heatmapHeight_;
+  if (this.data.options.showProfiles) {
+    transformTop += this.profileHeight_;
+  }
+  if (this.data.options.showGeneLabels) {
+    transformLeft += this.geneLabelWidth_ + this.LABEL_MARGIN;
+  }
+  else if (this.data.options.showProfiles) {
```

Comment:
https://google.github.io/styleguide/javascriptguide.xml?showone=Code_formatting#Code_formatting

or if this is still not convincing enough, then your favorite airbnb style guide too:

https://github.com/airbnb/javascript#16.2

:)